NEMO/static/nemo.js

```diff
@@ -96,6 +96,78 @@ function collapse_all_categories()
 	save_sidebar_state();
 }
 
+function get_qualified_tools_for_user(user_id) {
+	ajax_get("/api/users/"+user_id+"/", {}, (data) => {
+		localStorage.setItem("qualifiedTools", data.qualifications)
+	})
+}
```

Comment:
that part won't work, regular users don't have access to the API.
you might be able to use `{{ user.qualifications.all|json_search_base }}` either directly in calendar.html or by passing it to the JS function and grab the ids from the result instead

Reply:
Function was removed to use `retrieve_user_qualifications` instead, following your advice